frontend/src/components/button/index.js

```diff
@@ -0,0 +1,27 @@
+import styled from '@emotion/styled'
+import { fontSize, lineHeight, space, color, fontWeight } from 'styled-system'
+import tag from 'clean-tag'
+
+export const BaseButton = styled(tag.button)`
+  font-family: ${({ theme }) => theme.fontSans};
+  margin: 0;
+  border: 0;
+  &:focus {
+    outline: 3px solid #ffbf47;
+  }
+  ${fontSize};
+  ${lineHeight};
+  ${space};
+  ${color};
+  ${fontWeight};
+`
+
+BaseButton.propTypes = {
+  ...fontSize.propTypes,
+  ...lineHeight.propTypes,
+  ...space.propTypes,
+  ...color.propTypes,
+  ...fontWeight.propTypes,
+}
+
+export { Button } from './presets'
```

Comment:
The relationship between index.js and presets.js seems a little circular. index.js > BaseButton > presets.js > Button > index.js. Should we combine these into a single file?

Reply:
Maybe? @evadb , thoughts?

Reply:
In some cases with only one preset I think it makes more sense to combine everything. If we end up building a bunch of different button presets for whatever reason I can't imagine at the moment, I like the idea of having a separate file for organization